pkg/api/render.go

```diff
@@ -55,6 +57,15 @@ func (hs *HTTPServer) RenderToPng(c *m.ReqContext) {
 		return
 	}
 
+	if err != nil && err == rendering.ErrPhantomJSNotInstalled {
+		if strings.HasPrefix(runtime.GOARCH, "armv") {
```

Comment:
I think you need to check against `arm`, not `armv`.

See: https://golang.org/doc/install/source#environment